tests/testthat/dummy_packages/assignmentLinter/DESCRIPTION

```diff
@@ -7,3 +7,6 @@ Author: Who wrote it
 Maintainer: Who to complain to <yourfault@somewhere.net>
 Description: More about what it does (maybe more than one line)
 License: What license is it under?
+Suggests: 
+    testthat (>= 3.0.0)
+Config/testthat/edition: 3
```

Comment:
I used `usethis::use_testthat()` inside the `assignmentLinter` package to create a realistic case for doubly nested files.